src/pages/billing/List.js

```diff
@@ -328,7 +332,7 @@ export const BillingList = ({ queryParams: { selectedName } }) => {
           loadProjects()
         }
       }),
-      div({ style: { marginLeft: billingProjectListWidth, flexGrow: 1 } }, [Utils.cond(
+      div({ style: { flexGrow: 1, height: totalViewHeightMinusHeaderAndFooter, overflowY: 'scroll' } }, [Utils.cond(
```

Comment:
I am not sure which change might have broken it, but the sidebar scrolling is now totally not working for me.  The whole page is scrolling off of the header.